### PR TITLE
Add ability to set number of parallel runs (closes #27)

### LINKED ISSUE #27: Running tests on Jenkins: TypeError: Cannot read property 'message' of undefined
When running our tests on our Jenkins, I get an error

    /var/lib/****/node_modules/browserstack-local/lib/Local.js:61
              callback(new LocalError(data['message']['message']));
                                                     ^
    
    TypeError: Cannot read property 'message' of undefined
        at /var/lib/****/node_modules/browserstack-local/lib/Local.js:61:50
        at ChildProcess.exithandler (child_process.js:267:7)
        at emitTwo (events.js:126:13)
        at ChildProcess.emit (events.js:214:7)
        at maybeClose (internal/child_process.js:925:16)
        at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)

The issue here is, that the response of the `childProcess.execFile` in the `browserstack-local` package is empty. `error` is `null` and `stdout` and `stderr` are both empty strings. That's why `data` is an empty object.

I tried investigating what is going on, but I did not find a solution.

The binaryArgs for our run were:

    [ '--daemon',
      'start',
      '--log-file',
      '/dev/null',
      '--key',
      '****',
      '--local-identifier',
      1523874577640,
      '--enable-logging-for-api' ]

If I try to provide a logfile, it will be empty.

We do not use a proxy.

If I run the tests locally on my machine (with the same credentials), I do not get an error. The tests start normally.

Did anybody ever have a similar issue or does anybody have an idea what might be going on? Any help would be appreciated!

### PR DESCRIPTION
**Pull request by Lukas-Kullmann**

### CODE REVIEW

**Comment by AndreyBelym** [comment]
@Lukas-Kullmann thanks for your effort! Could you please clarify one thing: you have to set this flag even if you want to run just one job? Do you need to specify the exact number of jobs, if you want to run multiple jobs, or you can specify smaller or bigger number?

**Comment by AndreyBelym** [comment]
I'm going to accept the PR, however I'm quite afraid to break working builds, because I suspect that the default value of `--parallel-runs` [is not 1](https://github.com/DevExpress/testcafe-browser-provider-browserstack/issues/27#issuecomment-381623669). So I would avoid setting this flag if no corresponding environment variable was specified.

**Comment by Lukas-Kullmann** [comment]
@AndreyBelym Yes, in our environment I need to set any number to be able to connect to Browserstack, even if I just want to run one job.

It worked for me, but I'm not sure what happens if you have a subsciption for more than one multiple build or if you try to run more than one parallel build. Since our subscription is (still) very limited, this would not be a usecase for us anyway. Maybe you could try it with your build process and give me some feedback how it went. :)

**Comment by AndreyBelym** [comment]
Looks like the flag doesn't make sense in my case, I can run 3 browsers at the same time with `--parallel-runs 1` and vice-versa.

**Comment by AndreyBelym** [comment]
@VasilyStrelyaev, could you please check https://github.com/DevExpress/testcafe-browser-provider-browserstack/pull/28/files#diff-04c6e90faac2675aa89e2176d2eec7d8R17?

File: README.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Before using this plugin, save the BrowserStack username and access key to envir
 
 Project name and build name will be displayed in BrowserStack if you set the environment variables `BROWSERSTACK_PROJECT_NAME` and `BROWSERSTACK_BUILD_ID`.
 
+If you have troubles with starting multiple browsers at once, or `browserstack-local` related errors like [#27](https://github.com/DevExpress/testcafe-browser-provider-browserstack/issues/27), 
+try to set the `BROWSERSTACK_PARALLEL_RUNS` environment variable to the number of simultaneous browsers you want to run, or to 1 if you want to run just one browser.
```
Comment (VasilyStrelyaev) [inline README.md]: If you have troubles starting multiple browsers at once, or get `browserstack-local` related errors like [#27](https://github.com/DevExpress/testcafe-browser-provider-browserstack/issues/27), 
try setting the `BROWSERSTACK_PARALLEL_RUNS` environment variable to the number of browsers you want to run simultaneously, or to 1 if you want to run just one browser.

**Pull request merged**